evap/results/templates/results_index_evaluation_impl.html

```diff
@@ -50,10 +50,13 @@
         </div>
     {% endif %}
     {% if evaluation.is_single_result %}
-        <div data-col="participants" class="text-center"><span class="fas fa-user"></span>&nbsp;{{ evaluation.single_result_rating_result.count_sum }}</div>
+    <!-- data-order="{{ 1 }}" is used to show the singe result ratings on the bottom when sorting desc / top when asc -->
+    <div data-col="participants" class="text-center" data-order="{{ 1 }}"><span class="fas fa-user"></span>&nbsp;{{ evaluation.single_result_rating_result.count_sum }}</div>
```

Comment:
```suggestion
        {# data-order="-1" sorts single results before all other results #}
        <div data-col="participants" class="text-center" data-order="-1"><span class="fas fa-user"></span>&nbsp;{{ evaluation.single_result_rating_result.count_sum }}</div>
```

Reply:
(It may also be desirable that single results are still ordered within their "category", currently their order would be random I guess?)

Reply:
Agree. This may change completely if we would change to "total participants" instead of percentage anyways

Reply:
We can keep the order as is for now.